nestkernel/conn_parameter.h

```diff
@@ -69,8 +69,8 @@ class ConnParameter
    * @param rng   random number generator pointer
    * will be ignored except for random parameters.
    */
-  virtual double value_double( index, index, librandom::RngPtr& ) const = 0;
-  virtual long_t value_int( index, index, librandom::RngPtr& ) const = 0;
+  virtual double value_double( thread, librandom::RngPtr& ) const = 0;
+  virtual long_t value_int( thread, librandom::RngPtr& ) const = 0;
```

Comment:
I agree with Jochen. We discussed at an earlier point that we may want to introduce a general interface for parameter-generating functions which should receive `sgid` and `tgid` as input. If we want to retain this possibility, we need to pass them in, therefore we should pass them and the target thread in addition.